### ISSUE TITLE
ObjectManager support

### ISSUE THREAD

**Issue by muka** [godbus/dbus#57]
Hi,
is the support for org.freedesktop.DBus.ObjectManager planned? 

Thanks
Luca

**Comment by philips** [comment]
Nothing planned. Want to help?

**Comment by muka** [comment]
Yes, actually I've started playing on it already (https://github.com/muka/dbus/tree/object_manager) but looks like I'll need some time to get confident with the library code. Any input is welcome :)

**Comment by philips** [comment]
cc @jsouthwork

**Comment by philips** [comment]
cc @jsouthworth

**Comment by muka** [comment]
Closing for now, will reopen when I have it working

**Comment by muka** [comment]
Hi @jsouthworth, I have some code on the topic here 

- https://github.com/muka/go-bluetooth/blob/master/api/manager.go
- https://github.com/muka/go-bluetooth/blob/master/bluez/profile/ObjectManager.go

Do you think it may be useful to port the code in the library, after proper refactoring and review ?

Best

**Comment by jsouthworth** [comment]
That looks useful to me.

**Comment by muka** [comment]
Still alive, work is going on for server side also (https://github.com/muka/go-bluetooth/blob/master/service/ObjectManager.go) 

Will provide a PR once done
